examples/open_loop_control_fetch/run.py

```diff
@@ -49,12 +75,12 @@
         cv2.imshow("camera.final_color", obs["camera.final_color"])  # note that spear.Env returns BGRA by default
         cv2.waitKey(1)
 
-    for i,row in df.iterrows():
-        action = { k:np.array([v], dtype=np.float32) for k,v in row.to_dict().items() }
+    for i, row in df.iterrows():
+        action = {k: np.array([v], dtype=np.float32) for k, v in row.to_dict().items()}
         obs, reward, done, info = env.step(action=action)
 
         if not args.benchmark:
-            cv2.imshow("camera.final_color", obs["camera.final_color"]) # note that spear.Env returns BGRA by default
+            cv2.imshow("camera.final_color", obs["camera.final_color"])  # note that spear.Env returns BGRA by default
             cv2.waitKey(1)
 
         if args.save_images:
```

Comment:
- use `plt.imsave` for consistency with our other examples
- use `os.path.realpath(os.path.join(...))` for consistency with our other code
- put the `if args.save_images` inside the `if not args.benchmark` if statement for consistency with our other examples